browser/src/control/Control.ContextToolbar.ts

```diff
@@ -282,6 +290,38 @@ class ContextToolbar {
 		];
 	}
 
+	insertAdditionalContextButton(button: any) {
+		for (const i in this.additionalContextButtons) {
+			const item = this.additionalContextButtons[i];
+			if (item.id === button.id) return;
+		}
+
+		if (this.additionalContextButtons.length === 0) {
+			this.additionalContextButtons.push({
+				type: 'separator',
+				id: 'home-insertannotation-break',
+				orientation: 'vertical',
+			} as SeparatorWidgetJSON);
+		}
+
+		const isUnoCommand =
+			button.unoCommand && button.unoCommand.indexOf('.uno:') >= 0;
+		if (button.unoCommand && !isUnoCommand)
+			button.unoCommand = '.uno:' + button.unoCommand;
+
+		const contextButton = {
+			id: button.id,
+			type: button.type ? button.type : 'toolitem',
```

Comment:
```suggestion
			type: button: 'toolitem',
```
type is not supposed to be configurable.